components/support/nimbus-fml/src/frontend.rs

```diff
@@ -131,6 +131,8 @@ pub(crate) struct FeatureBody {
     #[serde(alias = "defaults")]
     #[serde(skip_serializing_if = "Option::is_none")]
     pub(crate) default: Option<Vec<DefaultBlock>>,
+    #[serde(default)]
+    pub(crate) allow_coenrollment: bool,
```

Comment:
I think you should be able to skip this if it's `false`.

Edit: you can, but it's a little clunky:
```suggestion
    #[serde(default)]
    #[serde(skip_serializing_if = "std::ops::Not::not")]
    pub(crate) allow_coenrollment: bool,
```